core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -176,11 +176,27 @@ void create(String tableName, NewTableConfiguration ntc)
    *
    * @param tableName
    *          Name of a table to create and import into.
-   * @param importDir
-   *          Directory that contains the files copied by distcp from exportTable
+   * @param importDirs
+   *          A comma-delimited list of directories containing the files copied by distcp from
+   *          exportTable
    * @since 1.5.0
+   *
+   * @deprecated since 2.1.0 use {@link #importTable(String, Set)} instead.
+   */
+  @Deprecated(since = "2.1.0", forRemoval = true)
+  void importTable(String tableName, String importDirs)
```

Comment:
I think this should be singular "importDir" now (or whatever it was called before the changes in #1060 )